05/helpers/formSender.js

```diff
@@ -0,0 +1,41 @@
+import { init, send } from 'emailjs-com';
+import account from './account';
+import ACTIONS from './actions';
+import fields from './formFields';
+import { convertArrToObj } from './helpersFunctions';
+
+function prepareDataToSend(data) {
+    const dataArr = fields.map((field) => {
+        const { name } = field;
+        const { value } = data[name];
+        if (value.length !== 0) {
+            return {
+                [name]: value,
+            };
+        }
+        return null;
+    });
+    return convertArrToObj(dataArr);
+}
+
+const info = {
+    succes: 'Message sent!',
+    failed: 'Message not sent.',
+};
+
+export default function sendMessage(data, dispatch, setInfo) {
+    init(account.userID);
+    const msgData = prepareDataToSend(data);
+    send(account.serviceID, account.templateID, msgData)
```

Comment:
Przydałby się jakiś "loader" przy wysyłaniu wiadomość.
Najlepiej też zrobić pola formularza jako disabled przy wysyłaniu.
Po zakończeniu je ponownie odblokować. To powoduje, że użytkownik nie zmodyfikuje pola w trakcie i wie, że wysyłanie trwa oraz kiedy się zakończy ;) ;)